libs/wire-api-federation/src/Wire/API/Federation/BackendNotifications.hs

```diff
@@ -51,6 +54,7 @@ instance FromJSON BackendNotification where
       <*> o .: "targetComponent"
       <*> o .: "path"
       <*> (RawJson . TL.encodeUtf8 <$> o .: "body")
+      <*> o .: "requestId"
```

Comment:
This is dangerous. The parser needs to be backwards compatible so any old things which are queued will still get processed after a restart.